sdk/servicebus/service-bus/test/batchReceiver.spec.ts

```diff
@@ -795,7 +795,9 @@ describe("batchReceiver", () => {
     // See https://github.com/Azure/azure-service-bus-node/issues/31
     async function testSequentialReceiveBatchCalls(useSessions?: boolean): Promise<void> {
       const testMessages = useSessions ? messageWithSessions : messages;
-      await senderClient.sendBatch(testMessages);
+      for (const message of testMessages) {
```

Comment:
Seems like a good spot to use your batch API?

Reply:
I was wondering if it would be more lines to create the batch and try add the messages and then send the batch.
Updated now.